### PR TITLE
Draft proposal version for the Python Software Foundation

### PR DESCRIPTION
**Pull request by Anavelyz**
This document contains the information that will be used to complete the PSF form.

### CODE REVIEW

File: psf_proposal.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+- FAQ & Grants Program Info *:
+
+    I have read and understand these pages (es una casilla de selección)
+    Note: Before completing this application, read the web pages pertaining to the PSF’s grants program, freely available tools to run virtual meetings, and understand how funding works:
+    https://www.python.org/psf/grants
+    https://www.python.org/psf/grants/free-resources/
+
+Date *: 
+MM/DD/YYYY
```
Comment (xmnlab) [inline psf_proposal.md]: Let's plan it to the middle of february

File: psf_proposal.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Provide the start date for your event/project
+Event’s/project’s website *:
+
+Requested Grant Amount (USD) *:
```
Comment (xmnlab) [inline psf_proposal.md]: 2k

File: psf_proposal.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+Date *: 
+MM/DD/YYYY
+Provide the start date for your event/project
+Event’s/project’s website *:
+
+Requested Grant Amount (USD) *: 
+
+Grant Amount Breakdown *:
+
+What amount are you requesting from the PSF? If the grant is funding several components of a project/event, please provide a breakdown of what the funding will be used for. Example: Our event is requesting $1500 total ($1000 for captioning/transcripts, $500 for improved internet bandwidth for speakers).
```
Comment (xmnlab) [inline psf_proposal.md]: I need to think a bit about that .. but I guess that almost all or all of the bugdet would be for development

File: psf_proposal.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+Budget *:
+Please include a link to a budget for the event or project work. This should show us an overview of all expenses and expected revenues. If you need to submit the budget via a document, please note that here and attach the document once you get to the "Other" section of this application.
```
Comment (xmnlab) [inline psf_proposal.md]: Maybe here we need to break it down by issues ... I will do it later today

File: psf_proposal.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+Objective *: 
+
+- What is the objective of your event/project?
+
+  Provide an up-to-date template with the best tools for creating Python packages.
```
Comment (xmnlab) [inline psf_proposal.md]:
```suggestion
  Improve a project template (for cookiecutter or cookieninja) that implements all (as much as possible) the best practices recomended by pyOpenSci, but not restricted to that, in order to offer all the current best practices approaches and tools as options, including CI, documentation, static analysis tools, etc. So anyone who wants to start a fresh project, would be able to start with a very good structure and workflow.

Additionally, part of the grant would help us to maintain it for some months, in order to update the pinnings for the libraries and some configuration changes.
```

File: psf_proposal.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+
+- Beneficiaries *:
+    Provide a description of the beneficiaries of this grant. If this request is for a sprint or virtual event, please tell us how many people are expected to attend. (e.g. if the grant is for a Python User Group (PUG) event, state the size of the group, the anticipated size of the event, and who in particular from the PUG benefits from the grant).
+    Regional Support
+    If you have funds remaining after your [event/development/etc], how do you plan to use them? For example: will your organization use remaining funding to help support regional Python efforts? Will remaining funds be saved for the next event? If so, please tell us about it!
```
Comment (xmnlab) [inline psf_proposal.md]: The remain funding would be used to maintain the the template  updated and, if it is possible, add new features.

File: psf_proposal.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+Budget *:
+Please include a link to a budget for the event or project work. This should show us an overview of all expenses and expected revenues. If you need to submit the budget via a document, please note that here and attach the document once you get to the "Other" section of this application.
+Objective *: 
+
+- What is the objective of your event/project?
+
+  Provide an up-to-date template with the best tools for creating Python packages.
+
+
+- Beneficiaries *:
```
Comment (xmnlab) [inline psf_proposal.md:68]: With this grant we plan to contract 2 people from our current group of collaborators

File: psf_proposal.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+- Link to Code of Conduct *: https://opensciencelabs.org/guidelines/mentoring/en/coc/ 
+
+    Provide a link to your code of conduct. We require that your code of conduct be listed on the event website as well. Meetup.com Requests: a link to, or the full text of your CoC must be in the "About" section of your Meetup page, or we cannot send you an invite to the PSF Meetup Group
+
+- Percentage of Python *: 100%
+    What percentage of your event/project will concentrate on Python?
+
+- Follow-up activities *:
+    Provide a description of the planned grant follow-up activities. If this grant is approved, we will send you a link to our report form. In addition to that, we want to know how post event/project information will be shared with the community. This could be blog postings, post-event write-ups, web pages, project updates, where recordings will be posted, etc.
```
Comment (xmnlab) [inline psf_proposal.md]: As a follow up activity, we plan to have a blog post and a webinar about all updates done via this grant.

**Review by xmnlab — COMMENTED** [review]
Added some comments

File: psf_proposal.md (hunk 1 above)
Comment (YurelyCamacho) [reply psf_proposal.md]: Y sobre esto:  Las solicitudes deben presentarse 6 semanas antes de la fecha de inicio del evento/proyecto.

File: psf_proposal.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+
+## Form
+
+- Contact
+
+  - Legal First Name*: Ivan
+
+  - Legal Last Name *: Ogasawara
+
+  - Nickname:
```
Comment (xmnlab) [inline psf_proposal.md]:
```suggestion
  - Nickname: xmnlab
```

File: psf_proposal.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+
+  - State/Province/Region: La Paz
+
+  - Region Other:
+
+  - Postal Code:
+
+  - Postal Code Suffix:
+
+  - Phone Number *:
```
Comment (xmnlab) [inline psf_proposal.md]:
```suggestion
  - Phone Number *: +591 69757089
```

**Review by xmnlab — COMMENTED** [review]
added info about my address.

**Review by xmnlab — APPROVED** [review]

**Comment by xmnlab** [comment]
thanks @Anavelyz

**Pull request merged**